app/src/main/java/com/jerboa/Utils.kt

```diff
@@ -1455,7 +1455,7 @@ fun Context.getInputStream(url: String): InputStream {
             Request.Builder()
                 .url(url)
                 .build(),
-        ).execute().body?.byteStream() ?: throw IOException("Failed to get input stream")
+        ).execute().body.byteStream()
```

Comment:
I added this in the last commit, because the compiler warned that the previous checks were pointless. But I can revert if there's something else going on.

Reply:
it was required when I remove the dependency for the custom log interceptor. But I kept it for now, so it isn't needed yet.  But I plan on removing that stuff since auth now is kept in the header. And I plan on moving rather to the LemmyAPI.